README.md

```diff
@@ -3,47 +3,81 @@
 This REDCap module provides options to adjust how validation is performed.
 
 
-## System-level configuration options
+## Field validation using conditional logic and regular expressions
 
-### Enable regular expression action tag
-Enable this to allow use of the **@REGEX** action tag.
+There are 3 action tags which can be used to provide additional field validation capabilities:
 
+* **@VALIDATE-LOGIC** &ndash; This can be used to supply conditional logic which must be valid in
+  order for the field value to be accepted.<br>
+  For example, if using an *email* and *confirm email* field, the confirm field could utilise the
+  action tag as follows:<br>
+  `@VALIDATE-LOGIC([email] = [email_confirm])`
+* **@VALIDATE-MESSAGE** &ndash; This can be used to supply a custom message to be displayed in
+  place of the standard message when a field validation error is triggered by the `@VALIDATE-LOGIC`
+  or `@REGEX` action tags.<br>
+  For example, a *confirm email* field might use:<br>
+  `@VALIDATE-MESSAGE="Sorry, the email addresses must match."`
+* **@REGEX** &ndash; This can be used to define a specific format as a regular expression which the
+  field value must satisfy.<br>
+  For example, an initials field which accepts 3 uppercase letters or 2 uppercase letters separated
+  by a hypthen could utilise the action tag as follows:<br>
+  `@REGEX="^[A-Z][A-Z-][A-Z]$"`<br>
```

Comment:
This is good to use@VALIDATE-LOGIC but it has to be the same form only.  Please indicate this.

Reply:
The `@VALIDATE-LOGIC` action tag is now fixed so it will work with fields on different forms.